protobuf/protoeditor-core/src/com/intellij/protobuf/bnf/proto.bnf

```diff
@@ -274,14 +274,18 @@ private meta BlockEntry ::= !'}' <<p>> { pin = 1 }
 /*
  * Syntax statement.
  */
-SyntaxStatement ::= syntax '=' StringValue ';'
+SyntaxType ::= syntax | edition
```

Comment:
Is it necessary to introduce a separate PSI element representing a keyword? If you don't mind, I'd rather make this rule private and make SyntaxStatement#getSyntaxType method return enum. Currently it would have two options: syntax, edition. WDYT?

Reply:
No, it shouldn't be necessary.  And making it private sounds good, since under editions nobody should really be checking syntax/edition.  Long-term, you'd want to look at the features to truly understand the files